widgetastic_manageiq/__init__.py

```diff
@@ -5513,6 +5529,50 @@ def get_progress_percent(self, plan_name):
         ]
         return dict(list(zip(desc, val)))
 
+    def schedule_cutover(self, plan_name, cancel=False, immediate=False, after_mins=5):
+        try:
+            el = self._get_card_element(plan_name)
+            cutover_button = self.browser.element(self.ITEM_CUTOVER_BUTTON_LOCATOR, parent=el)
+            wait_for(
+                lambda: cutover_button.is_enabled(),
+                delay=5,
+                num_sec=120,
+                message="waiting for cutover button to be enabled.",
+            )
+            cutover_button.click()
+            if immediate:
+                self.root_browser.click(self.IMMEDIATE_BTN)
+            else:
+                cutover_time_input = self.root_browser.element(self.ITEM_SCHEDULE_INPUT_LOCATOR)
+                cutover_time_input.click()
+                try:
+                    for i in range(after_mins):
+                        self.root_browser.click(self.ITEM_MODAL_MINUTE_INCREMENT_LOCATOR)
+                except NoSuchElementException:  # Throws this in FF browser
+                    self.logger.info(
+                        "Schedule using dateTimePicker failed, using datetime and timedelta."
+                    )
+
+                    current_time = datetime.strptime(
+                        cutover_time_input.get_attribute("value"), "%m/%d/%Y %I:%M %p"
+                    )
+                    cutover_time_input.clear()
+                    cutover_time_input.send_keys(
+                        datetime.strftime(
+                            current_time + timedelta(minutes=after_mins), "%m/%d/%Y %I:%M %p"
+                        )
+                    )
+                    # clicking on title to lose focus from dateTimePicker & activate schedule button
+                    self.root_browser.click('.//h4[@class="modal-title"]')
```

Comment:
@mnadeem92 : please create a Title_locator for this instead of using the xpath directly .